app/main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+import random
+import matplotlib.pyplot as plt
+
+
+def flip_coin():
+    cases = 10000
+    flips = 10
+
+    flip_dict = {0: 0, 1: 0,
+                 2: 0, 3: 0,
+                 4: 0, 5: 0,
+                 6: 0, 7: 0,
+                 8: 0, 9: 0,
+                 10: 0}
+
```

Comment:
And what if the range was 100? I think it's better to generate dict in required range.

Reply:
Ofc, but in our condition we have a static length of dict, it's not necessary. But thanks for your comment I'll change it.

Reply:
It is ok for few keys, but 10 is more than enough not to do it manually.